deployment/chainloop/templates/controlplane/deployment.yaml

```diff
@@ -91,6 +91,13 @@ spec:
             - name: gcp-secretmanager-serviceaccountkey
               mountPath: /gcp-secrets
             {{- end }}
+            {{- if (not (empty .Values.controlplane.customCAs)) }}
+            - name: custom-cas
+              # NOTE: /etc/ssl/certs already contains the system CA certs
+              # Let's use another known path https://go.dev/src/crypto/x509/root_linux.go
+              mountPath: /etc/pki/tls/certs
```

Comment:
I think you can still mount it in the standard /etc/ssl/certs folder. That's what I've done previously in similar changes. Other tools or libraries might not support this other path.

Reply:
A subpath within `/etc/ssl/certs/` won't work right?

Reply:
It doesn't seem to load recursively no :(

https://cs.opensource.google/go/go/+/master:src/crypto/x509/root_unix.go;l=62;drc=fe36ce669c1a452d2b0e81108a7e07674b50692a

Reply:
As long as the entire folder doesn't get overridden, you can have as many files as you want there. Just make sure of mounting the file, not the whole folder.

Reply:
Right, but the files are dynamic, so you can add as many certs as you want. I could definitely generate dynamically the mount points indeed. 

I'll do it in another patch

Reply:
Tracked https://github.com/chainloop-dev/chainloop/issues/965